### PR TITLE
Add account id to account name

### PR DESCRIPTION
**Pull request by bkochendorfer**
I am often confused searching the login page for the AWS Account ID that corresponds to the Role I want to select. This simply add the Account ID to the output for faster finding. Open to suggestions or feedback if folks have other workflows that solve this.

Command Line Version of this request:
https://github.com/mozilla-iam/mozilla-aws-cli/pull/241

### CODE REVIEW

**Review by gene1wood — APPROVED** [review]

**Review by duallain — APPROVED** [review]
Brett, this may not deploy on merge for you.  After it merges, feel free to ping me and I can help get it deployed. (on second thought it may not be in an account we have access to, but I can show you how to validate that at least).

**Comment by gene1wood** [comment]
>  this may not deploy on merge for you

Correct, deployment is via a CloudFormation stack update

https://github.com/mozilla-iam/federated-aws-rp#deploying

> it may not be in an account we have access to

Good point. So the docs say it's in `us-east-1` but not what account. Lemme see what accounts its in (dev and prod) and PR the README

**Comment by gene1wood** [comment]
Ok, #35 has details about the Mozilla deployment. Indeed, the dev instance is in `infosec-dev`. Either I can deploy a dev update after this is merged, or you can migrate the dev instance from `infosec-dev` to `mozilla-iam`. Pretty much you'll just need a domain name and an ACM cert in `mozilla-iam` for a dev deployment, that's it. But your call.

**Comment by bkochendorfer** [comment]
I'll give this a merge and if you don't mind @gene1wood could you do the `dev` deploy?

**Comment by gene1wood** [comment]
Deploy to dev complete, looks good to me

**Pull request merged**
